frontend/src/styles/vars.scss

```diff
@@ -15,7 +15,7 @@ $screens: (
     'xxl': $xxl,
 );
 $tiny_spaces: 0, 0.5, 1, 1.5, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20;
-$humongous_spaces: 24, 30, 32, 40, 50, 60, 80, 100, 120, 140, 160, 180, 200;
+$humongous_spaces: 24, 30, 32, 40, 50, 60, 70, 80, 100, 120, 140, 160, 180, 200;
```

Comment:
Could we use the `80` value here? It feels like we're adding a variable for a single use - maybe that should be a class

Reply:
I had tried the 80 and liked 70 better, hence the addition. I don't think it's too big of an issue though, as tailwind supports arbitrary values https://tailwindcss.com/docs/min-width#arbitrary-values. Meaning when we finally have real tailwind support we can remove all custom spaces and just use the tailwind default ones here.